sdk/core/build.gradle

```diff
@@ -0,0 +1,2 @@
+description = 'Core featureset of the Klaviyo SDK including SDK configuration, session tracking and analytics'
+evaluationDependsOn(':sdk')
```

Comment:
This declares that buildouts of this module depend on the SDK module (which is the higher level)